sigproc/inc/WireCellSigProc/OmnibusSigProc.h

```diff
@@ -236,8 +235,8 @@ namespace WireCell {
             std::string m_mp2_roi_tag;
 
 	    //Rebase waveforms for each channel of spesific wire-plane. 
-	    std::vector<int> m_rebase_planes; 
-            int m_rebase_nbins;
+	    std::vector<int> m_rebase_planes{-999}; 
```

Comment:
why initialized with a special number instead of an empty vector like suggested by @brettviren?

Reply:
It is just my preference, let me fix it